docs/collecting-data/code-generation/working-with-gtm/index.md

```diff
@@ -24,6 +28,36 @@ To generate code for usage in Google Tag Manager, you should use the option `Goo
     "language": "javascript-gtm"
 }
 ```
+</details>
+
+What we are going to showcase here is how you can setup a separate project that uses Snowtype to generate code for you Google Tag Manager tracking needs. _You can also use version control to better understand any changes you do across time._
```

Comment:
- "setup" should be "set up" in "how you can setup a separate project".
- "you Google Tag Manager tracking needs" should be "your Google Tag Manager tracking needs".
- "any changes you do across time" should be "any changes you make over time".
- "installed in your computer" should be "installed on your computer".
- Missing comma after "After that".
- "Change directory in the newly created folder" should be "Change directory to the newly created folder".
- "in which you should select 'Google Tag Manager' when prompted to select a tracker" could be made clearer with a slight rephrasing.

Reply:
Done all of them except the last one since I cannot think of something better 😂